platforms/nuttx/src/px4/stm/stm32_common/io_pins/io_timer.c

```diff
@@ -87,7 +87,9 @@ static int io_timer_handler7(int irq, void *context, void *arg);
  * taken into account
  */
 #if !defined(BOARD_PWM_FREQ)
-#define BOARD_PWM_FREQ 1000000
+// #define BOARD_PWM_FREQ 1000000
```

Comment:
This shouldn't stay commented out. Could the downscaling be something board-specific that breaks other boards?

Reply:
We could probably make this configurable. It's mostly for thruster based systems that require a smaller frequency